docs/apm/infrastructure.asciidoc

```diff
@@ -0,0 +1,13 @@
+[role="xpack"]
+[[infrastructure]]
+=== Infrastructure
+
+The *Infrastructure* tab provides information about the containers, pods, and hosts,
+that the selected service is linked to.
+
+[role="screenshot"]
+image::apm/images/infra.png[Example view of the Infrastructure tab in APM app in Kibana]
+
+IT ops and software reliability engineers (SREs) can use this tab
+to quickly find a service's underlying infrastructure resources when debugging a problem.
+Knowing what infrastructure is related to a service allows you to remediate issues by restarting, killing hanging instances, changing configuration, rolling back deployments, scaling up, scaling out, etc.
```

Comment:
👋 @MiriamAparicio, you were listed as the point of contact for the new infrastructure tab. Is there anything you would add or change to this?